packages/tool-cache/src/tool-cache.ts

```diff
@@ -206,6 +207,29 @@ export async function extractTar(
   return dest
 }
 
+/**
+ * Extract a xar compatible archive
+ *
+ * @param file     path to the archive
+ * @param dest     destination directory. Optional.
+ * @param flags    flags for the xar. Optional.
+ * @returns        path to the destination directory
+ */
+export async function extractXar(
```

Comment:
Can we add an example to the [readme](https://github.com/actions/toolkit/blob/master/packages/tool-cache/README.md)?